velox/exec/tests/utils/AggregationFuzzer.cpp

```diff
@@ -1604,19 +1604,48 @@ bool AggregationFuzzer::verifySortedAggregation(
     LOG(INFO) << "Verified results against reference DB";
   }
 
+  std::vector<PlanWithSplits> plans;
+  if (!groupingKeys.empty()) {
+    plans.push_back(
+        {PlanBuilder()
+             .values(input)
+             .orderBy(groupingKeys, false)
+             .streamingAggregation(
+                 groupingKeys,
+                 aggregates,
+                 masks,
+                 core::AggregationNode::Step::kSingle,
+                 false)
+             .planNode(),
+         {}});
+  }
+
   const auto inputRowType = asRowType(input[0]->type());
   if (isTableScanSupported(inputRowType)) {
     auto directory = exec::test::TempDirectoryPath::create();
     auto splits = makeSplits(input, directory->path);
 
-    std::vector<PlanWithSplits> plans;
     plans.push_back(
         {PlanBuilder()
              .tableScan(inputRowType)
              .singleAggregation(groupingKeys, aggregates, masks)
              .planNode(),
          splits});
 
+    if (!groupingKeys.empty()) {
+      plans.push_back(
+          {PlanBuilder()
+               .tableScan(inputRowType)
+               .orderBy(groupingKeys, false)
+               .streamingAggregation(
+                   groupingKeys,
+                   aggregates,
+                   masks,
+                   core::AggregationNode::Step::kSingle,
+                   false)
+               .planNode(),
+           splits});
+    }
     // Set customVerification to false to trigger direct result comparison.
     // TODO Figure out how to enable custom verify(), but not compare().
     testPlans(plans, false, {}, resultOrError, 1);
```

Comment:
Can you make sure we cll testPlans() even if table scan is not supported?

Reply:
Got it, I should notice this, moved it out of the if.